python/src/aiconfig/editor/client/src/themes/VSCodeTheme.ts

```diff
@@ -128,26 +149,27 @@ export const VSCODE_THEME: MantineThemeOverride = {
     },
 
     ".parametersContainer": {
-      maxWidth: "1014px",
+      width: "100%",
       maxHeight: "-webkit-fill-available",
       margin: "16px auto",
```

Comment:
Probably want 36px margin left to match https://github.com/lastmile-ai/aiconfig/pull/1077